air-quality-ui/system_tests/ui/city_page.spec.ts

```diff
@@ -346,3 +347,110 @@ test.describe('City graph snapshots', () => {
     expect(chartShot).toMatchSnapshot('rio-so2-graph-without-centro.png')
   })
 })
+
+test.describe('Charts are visible in immediate view', () => {
```

Comment:
I think it would be clearer to name this something like 'On a 1920x1080 display, all charts can be displayed on a single page with no scrolling' and to deliberately set the viewport size before each test to make this clear (even if it does it by default)